README.md

```diff
@@ -73,6 +73,13 @@ substituting `<REPORT CONFIG FILENAME>` for the report config you wish to use fr
 This may take some time and you'll be informed of progress. Once complete, the reports will be saved in the `data`
 directory.
 
+For some reports, some post-processing needs to be done before sharing with the Accessibility team. Where a report
+requires this, then you will find it under `notebooks/<report_name>_postprocess/py`.
+
+For example, if you want to create the `src/report_generators/attachment_type_report_generator.py` report, then run this
```

Comment:
You could probably use the config file and add in a conditional into `run_accessibility_reports.py`. Also means we don't create multiple ways of generating reports.

Reply:
Yes that would be the nicer way to do it but since we need to do some post-processing on the entire `csv` rather than line-by-line, I'm not sure if it can fit neatly within the `run_accessibility_reports.py` framework.

Reply:
Yes, sorry I was being vague and unclear: we could add to the `__main__` to run the post-process if the option in the config file for that report was set to run that report. Or would that not work? That way the documentation and ways of running it can be consistent for the user.

Reply:
Outside of the loops

Reply:
Makes sense, I've added it now but not sure if I added it correctly. Will appreciate your help. 🙂 Unfortunately, I can't test because I need to download the mirror and since I'm using my personal macbook, I have no memory for it (serves me right for buying the lowest memory version!).

Reply:
Ah I don't have the files either... it'll take a while to download. I'll see if I can kick this off.